packages/material-ui-lab/src/Autocomplete/Autocomplete.js

```diff
@@ -287,6 +288,7 @@ const Autocomplete = React.forwardRef(function Autocomplete(props, ref) {
       >
         {renderInput({
           ref: setAnchorEl,
+          id,
```

Comment:
Used by TextField to link the helper text, input, and label together. Error reported by Wave and https://validator.w3.org/.